pkg/asset/machines/master.go

```diff
@@ -81,7 +82,7 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 		if err != nil {
 			return errors.Wrap(err, "failed to create master machine objects")
 		}
-		aws.ConfigMasters(machines, ic.ObjectMeta.Name)
+		aws.ConfigMasters(machines, ic.ObjectMeta.Name, mpool.InstanceType)
```

Comment:
why is this required?

Reply:
@wking ^ this is what I get :)

Reply:
> why is this required?

Ah, sorry about that.  Yeah, we can probably drop this, since `aws.provider` should be applying this for us.

Reply:
@smarterclayton, can you unwind is change with:

```diff
diff --git a/pkg/asset/machines/aws/machines.go b/pkg/asset/machines/aws/machines.go
index 865d40a..7e04a99 100644
--- a/pkg/asset/machines/aws/machines.go
+++ b/pkg/asset/machines/aws/machines.go
@@ -120,12 +120,10 @@ func tagsFromUserTags(clusterID, clusterName string, usertags map[string]string)
 	return tags, nil
 }
 
-// ConfigMasters sets the PublicIP flag, bumps the instance type, and
-// assigns a set of load balancers to the given machines
-func ConfigMasters(machines []clusterapi.Machine, clusterName string, instanceType string) {
+// ConfigMasters sets the PublicIP flag and assigns a set of load balancers to the given machines
+func ConfigMasters(machines []clusterapi.Machine, clusterName string) {
 	for _, machine := range machines {
 		providerSpec := machine.Spec.ProviderSpec.Value.Object.(*awsprovider.AWSMachineProviderConfig)
-		providerSpec.InstanceType = instanceType
 		providerSpec.PublicIP = pointer.BoolPtr(true)
 		providerSpec.LoadBalancers = []awsprovider.LoadBalancerReference{
 			{
diff --git a/pkg/asset/machines/master.go b/pkg/asset/machines/master.go
index f4c3806..2c782ac 100644
--- a/pkg/asset/machines/master.go
+++ b/pkg/asset/machines/master.go
@@ -82,7 +82,7 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 		if err != nil {
 			return errors.Wrap(err, "failed to create master machine objects")
 		}
-		aws.ConfigMasters(machines, ic.ObjectMeta.Name, mpool.InstanceType)
+		aws.ConfigMasters(machines, ic.ObjectMeta.Name)
 
 		list := listFromMachines(machines)
 		raw, err := yaml.Marshal(list)
```

I'm testing that now to make sure it still works...